tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -1148,7 +1148,22 @@ async def test_offer_endpoints(wallet_rpc_environment: WalletRpcTestEnvironment)
     assert id == offer.name()
     id, advanced_summary = await wallet_1_rpc.get_offer_summary(offer, advanced=True)
     assert id == offer.name()
-    assert summary == {"offered": {"xch": 5}, "requested": {cat_asset_id.hex(): 1}, "infos": driver_dict, "fees": 1}
+    assert summary == {
+        "offered": {"xch": 5},
+        "requested": {cat_asset_id.hex(): 1},
+        "infos": driver_dict,
+        "fees": 1,
+        "valid_times": {
+            "max_blocks_after_created": None,
```

Comment:
these relative time locks; they are relative a specific coin. I assume an offer can contain multiple coins with (potentially) separate relative time locks.

At this point, when we have the offer file, offering concrete coins, we (can) know their creation times and heights. I think these fields could be boiled down to only the most restrictive *absolute* fields.

i.e. "max_height", "min_height", "max_seconds", "min_seconds"

I don't think you have enough information to interpret the relative timestamps in this RPC